msibi/optimize.py

```diff
@@ -120,11 +120,16 @@ def add_bond(self, bond):
     def add_angle(self, angle):
         self.angles.append(angle)
 
+
     def optimize(
         self,
         n_iterations=10,
-        n_steps=1e6,
         start_iteration=0,
+        n_steps=1e6,
+        integrator="hoomd.md.integrate.nvt",
+        integrator_kwargs = {"tau": 0.1},
```

Comment:
Using a dict as a default argument is dangerous. Check out this example
```
def f(value, key, hash={'tau':1}):
    hash[value] = key
    return hash

print(f('tau', 3))
print(f('b', 2))
```
output:
```
{'tau': 3}
{'tau': 3, 'b': 2}
```
If multiple calls are made to this function in the same kernel, it will continue to add to the same dictionary which is usually not desired. [This article explains more.](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html)